ui/src/components/TaskDetailsCard/index.jsx

```diff
@@ -93,6 +94,19 @@ export default class TaskDetailsCard extends Component {
         }),
       })
     ),
+    dependents: arrayOf(
+      shape({
+        taskId: string,
+        tasks: arrayOf(
+          shape({
+            status: shape({
+              state: string,
+            }),
+            task,
+          })
+        ),
+      })
+    ),
```

Comment:
`utils/prop-types` exports a bunch of props. We can probably just do `dependents: arrayOf(task)` where task is exported from that file.